src/database/schema/ChatSchema.ts

```diff
@@ -317,7 +319,9 @@ class ChatSchema implements BaseDbSchema {
       console.log(e);
     }
     try {
-      attachmentJson = JSON.stringify(json?.attachments);
+      if (json?.message_type !== 'reply_prompt') {
+        attachmentJson = JSON.stringify(json?.attachments);
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that `attachmentJson` has a default value to prevent potential errors when it's used later in the code.
```diff
+    let attachmentJson = null;
     if (json?.message_type !== 'reply_prompt') {
       attachmentJson = JSON.stringify(json?.attachments);
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->